### PR TITLE
allow no authentication smtp email delivery

### PR DESCRIPTION
**Pull request by pigay**

### CODE REVIEW

File: templates/home/redmine/redmine/config/configuration.yml.j2 [hunk 1]
```diff
@@ -3,15 +3,17 @@ default:
   # Outgoing emails configuration
   email_delivery:
 {% if redmine_enable_smtp_email %}
-    delivery_method: smtp
+    delivery_method: :smtp
```
Comment (pigay) [inline templates/home/redmine/redmine/config/configuration.yml.j2:6]: "smtp" alone is not a valid option (https://www.redmine.org/projects/redmine/wiki/EmailConfiguration#delivery_method)

**Review by bngsudheer — COMMENTED** [review]
Why are we checking for != 'none'?

**Comment by pigay** [comment]
It's just a keyword one can use to configure no authentication method.

Should I document this keyword in `README.md`?

**Review by bngsudheer — CHANGES_REQUESTED** [review]
I would do ```{% if redmine_smtp_settings_authentication %}```  instead of comparing to the string 'none'

**Comment by bngsudheer** [comment]
Yes, please document the changes.

**Comment by pigay** [comment]
Here it is.

**Comment by bngsudheer** [comment]
Apparently, the tests broke

**Comment by pigay** [comment]
Tests broke? Are they visible in github? I can't see the CI/CD.

[Edit] ok got it. I take a look at `tests/test.sh`...

**Comment by pigay** [comment]
I tried to run the test on both `v2.1.1` and `v2.2.0` but it fails with this message:
```
fatal: [localhost]: FAILED! => {"changed": false, "msg": "Could not find the requested service firewalld: host"}
```
Maybe a problem with my version of ansible?

Anyway, what is the error that breaks the tests?

**Pull request merged**